src/vaadin-menu-bar-buttons-mixin.html

```diff
@@ -211,6 +211,8 @@
         if (this.theme && this.theme !== '') {
           button.setAttribute('theme', this.theme);
         }
```

Comment:
This can be removed because we have `this._setButtonTheme(button, this.theme);` line below.

Reply:
Removed.